### PR TITLE
Support multiple scales.

### PR DESCRIPTION
**Pull request by wcarss**
In reference to https://github.com/irshadshalu/music-grid/issues/12, adds a handful of scale options in a select dropdown (also tried a range -- nothing better came to mind) to choose from and widens the grid by 2 columns to support a full chromatic scale. Encodes the scale in a new hash param, also reads one if present, and assumes classic if none found.

Existing compositions should be unaffected, but visiting them will yield a URL with the scale included.

Also, I *think* this will merge cleanly with the play_row_on_tap, but haven't had a chance to try the one atop the other yet.

### CODE REVIEW

File: src/Music.svelte [hunk 1]
```diff
@@ -6,11 +6,43 @@
 	let recorder;
 	let destinationStream;
 
-	let notes = [
-		'B3', 'C#4', 'F#4', 'G#4',
-		'C#5', 'D#5', 'E5', 'G#5',
-		'B5', 'C#6', 'F#6', 'G#6'
-	];
+	let scales = {
+		classic: [
```
Comment (irshadshalu) [inline src/Music.svelte:10]: Just curious why it has to be 14?

**Comment by irshadshalu** [comment]
I just merged this and #26 and tested.. unfortunately, there is some issue in the styling when the number of rows is increased: you can test at http://musicgrid.irshadpi.me/ . 


I don't know if the number of columns or the play row design has anything to do with it.

File: src/App.svelte [hunk 2]
```diff
@@ -128,6 +132,7 @@
 <style>
 	table {
 		background: black;
+		transform: scale(0.9);
```
Comment (irshadshalu) [inline src/App.svelte:135]: Why is this necessary?, is this a good idea?

Comment (irshadshalu) [reply src/App.svelte:135]: I think this is the cause for the issue.

Comment (wcarss) [reply src/App.svelte:135]: ahhh, I understand how that could cause the styling issue during resizing.

The transform is added as a simple way to maintain the styling of everything else but support more notes, because a chromatic scale that can't cover an octave is a little useless, and there's not enough room on most phones to display > 12 notes and the 'play row on tap' button.

Comment (wcarss) [reply src/App.svelte:135]: If you'd rather there just be 12 notes and to remove this scale, make sure to remove the _first two_ notes from the "classic" scale to preserve old compositions

Comment (wcarss) [reply src/App.svelte:135]: https://github.com/irshadshalu/music-grid/pull/31 was thrown up in a hurry, but should resolve the issue

Comment (wcarss) [reply src/App.svelte:135]: actually, just adding `transform-origin: top;` fixes the issue too, and allows for the scaling.

Comment (irshadshalu) [reply src/App.svelte:135]: I think keeping the 12 columns is better, 14 columns even after scaling looks very small when tested in another phone.

Comment (wcarss) [reply src/App.svelte:135]: sure! I'll change #31 back to 12 rows and no scaling.
